rfc/000-TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+# Proposal Name
+
+- Authors:
+  - [First Last](https://github.com/username)
+  - [First Last](https://github.com/username)
+
+## Overview
+
+A short summary of the propsal contents.
+
+## Problem Statement
+
+A description of the problem this proposal solves.
+
```

Comment:
Potentially, also requirements section. In my experience it's good to lay down assumptions one is working with.
On the other end, maybe some requirements are confidential (paying) customer ones that cannot be shared.

Reply:
Good idea. I assume you would consider this a top-level heading, following the Problem Statement?

Reply:
Yup, thanks!

Reply:
So I tried writing such a section, but I struggled a bit putting it down. This RFC tackles a UX-related topic. Is this a perhaps a subsection of the implementation section, when we go into the technical details?

Reply:
For UX aspects, I think this can be centered around the persona targeted by the change and their critical user journeys.

For example here I think it's a determinant aspect that the pipelines can only be updated by the clients. Maybe worth putting in this section and linking to the discussion in the relevant section as to why.

You could also list some important non-requirements / non-scope explicitly. For example here I think the python API is not really in scope.

Reply:
This is outdated (as been integrated in the separate PR), feel free to close this thread.